pywwt/annotation.py

```diff
@@ -230,3 +231,44 @@ def _on_trait_change(self, changed):
             changed['new'] = changed['new'].value
 
         super(Line, self)._on_trait_change(changed)
+
+class CircleCollection():
+    
+    def __init__(self, parent, points, **kwargs): # save args, make first circles
+        if len(points) <= 1e4: # subject to change
+            self.points = points
+        else:
+            raise IndexError('too many points') # correct error?
+        self.parent = parent
+        self.collection = []
+        self.gen_circles(self.points, **kwargs)
+        
+    def __setattr__(self, name, value):
+        if name in [key for key in Circle.__dict__ if not key.startswith('_')]:
+            for elem in self.collection:
+                setattr(elem, name, value)
+                #print(elem.__dict__['_trait_values'][name])
+        else:
+            self.__dict__[name] = value
+
+    def __getattr__(self, name):
+        if name in [key for key in Circle.__dict__ if not key.startswith('_')]:
+            values = []
+            for elem in self.collection:
+                attr = getattr(elem, name)
+                if attr not in values:
+                    values.append(attr)
+            return values
+        else:
+            raise AttributeError('CircleCollection object has no attribute '+repr(name))
+        
+    def gen_circles(self, points, **kwargs):
```

Comment:
Since I don't think ``gen_circles`` is meant to be used, maybe make it private, i.e. ``_gen_circles``